public/.well-known/apple-app-site-association.json

```diff
@@ -0,0 +1,14 @@
+{
+  "applinks": {
+    "apps": [],
+    "details": [
+      {
+        "appID": "DQ48D9BF2V.org.cru.mpdx",
+        "paths": [
+          "/contacts/*",
```

Comment:
I'm not sure what this does, but our contact links now look like `/accountLists/:id/contacts/*`

Reply:
Could it be URLs on the mobile apps?

Reply:
I'm guessing it makes clicking on a link (like in an email) open up the MPDX app.

Reply:
On MPDX Web, there isn't a `/auth/mobile` page

Reply:
@frett Could you speak into this? The `paths`, is it paths on a phone or on the web?

Reply:
the paths property are the url paths that the app will handle deep links for.

I made a comment elsewhere we probably need to update our deep link handling in the app anyways to support the new accountList URLs, but on Android that's a bit more complicated because it could switch to a different account list which will cause the app to wipe all local data currently.

Reply:
We will want to support the new url structure in the mobile apps so that deep links open correctly in the app.

The inclusion of the new deep link handling is what should cause the mobile team to submit a PR adding the new URL patterns we are handling to the appropriate files here.